component/src/main/java/org/wso2/carbon/extension/analytics/receiver/rabbitmq/internal/util/RabbitMQEventAdapterConstants.java

```diff
@@ -0,0 +1,79 @@
+/*
+*  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.wso2.carbon.extension.analytics.receiver.rabbitmq.internal.util;
+
+/**
+ * This class represents RabbitMQ specific Constants.
+ */
+public final class RabbitMQEventAdapterConstants {
+
+    private RabbitMQEventAdapterConstants() {
+    }
+
+    public static final String ADAPTER_TYPE_RABBITMQ = "rabbitmq";
+    public static final String RABBITMQ_SERVER_HOST_NAME = "hostname";
```

Comment:
Good to have javaDocs describing the role of each variable. Please fix these type of issues throughout in all your PR's